src/views/components/collection-links.html

```diff
@@ -3,7 +3,7 @@
 
 	{{#each links}}
 	<div class="collection-link{{#if ~/combined && !@index && links.length > 1}} combined{{/if}}">
-		<a target="_blank" href="{{text}}">{{html}}</a>
+		<a href="{{text}}" rel="noopener" target="_blank">{{html}}</a>
```

Comment:
Does this point to an external URL? If not we can drop the noopener addition

Reply:
No, that's our page.